src/Libraries/RevitNodes/Elements/Views/View.cs

```diff
@@ -245,14 +246,42 @@ public Revit.Filter.OverrideGraphicSettings FilterOverrides(Revit.Filter.Paramet
             return new Revit.Filter.OverrideGraphicSettings(overrides);
         }
 
-        #endregion
-
-        #region View Templates
-
         /// <summary>
-        ///     Checks if View is a View Template.
+        ///     Get the elements contained in the view of the specified category. 
         /// </summary>
-        /// <returns></returns>
+        /// <param name="category">Element Category</param>
```

Comment:
There is an existing node called "All Elements of Category" under Revit->Selection, if we want to add the node, it should also under Revit->Selection for consistance, maybe just add another "All Elements of Category" node which take a view and a category as parameters.
And there is a package, archi-lab.net, which also provide a node Select.ByCategoryAndView. 
@QilongTang, do you have any suggestions?

Reply:
@AndyDu1985  OK, after posting this it came to my attention the possibility of what you suggest. FYI, this PR's purpose is to provide an internal alternative to the ARchilab node as our team needs it internal to ship a script inside the box of revit.. I will go forwarard with your proposed changes.